EduPerfTests/Performance.cs

```diff
@@ -87,34 +64,35 @@ public void JetStream(Browser browser, RemoteWebDriver driver, int iterations)
                     Thread.Sleep(10000);
                 }
                 var result = driver.FindElementById("results-cell-geomean").Text.Substring(0, 6);
-                logFile.WriteLine(string.Format("JetStream,{0},{1},{2},", browser, result, i + 1));
+                _perfLog.WriteToLog($"JetStream,{browser},{result},{i + 1},");
             }
         }
 
         public void WebXPRT(Browser browser, RemoteWebDriver driver, int iterations)
         {
-            for (int i = 0; i < iterations; i++)
+            for (var i = 0; i < iterations; i++)
             {
                 driver.Url = "http://www.principledtechnologies.com/benchmarkxprt/webxprt/2015/v19982/";
                 driver.FindElementById("imgRunAll").Click();
-                while (driver.Url != "http://www.principledtechnologies.com/benchmarkxprt/webxprt/2015/v19982/results.php?c=0")
+                while (driver.Url !=
+                        "http://www.principledtechnologies.com/benchmarkxprt/webxprt/2015/v19982/results.php?c=0")
                 {
                     Thread.Sleep(10000);
                 }
                 var result = driver.FindElementByCssSelector(".resultsOval>.scoreText").Text;
-                logFile.WriteLine(string.Format("WebXPRT,{0},{1},{2},", browser, result, i + 1));
+                _perfLog.WriteToLog($"WebXPRT,{browser},{result},{i + 1},");
             }
         }
 
         public void OORTOnline(Browser browser, RemoteWebDriver driver, int iterations)
```

Comment:
Let's take a stab at it later. There is more to do here for everything. I think we should get this in and running regularly first, then design some improvements/changes for the summer.